odoo/addons/test_new_api/tests/test_new_fields.py

```diff
@@ -573,16 +573,6 @@ def test_13_inverse_access(self):
         with self.assertRaises(AccessError):
             foo.with_user(user).display_name = 'Forbidden'
 
-    def test_13_inverse_access(self):
```

Comment:
seems not related to the current PR ;)

Reply:
it is, this patch detects function/method/etc redefinitions, this test is defined twice, so only the second one sequentially is every executed (in this case both are identical so I just removed one of them), probably a forward-port bug